dot/rpc/modules/childstate_test.go

```diff
@@ -130,6 +131,61 @@ func TestGetStorageHash(t *testing.T) {
 	}
 }
 
+func TestGetChildStorage(t *testing.T) {
+	mod, blockHash := setupChildStateStorage(t)
+	randomHash, err := common.HexToHash(RandomHash)
+	require.NoError(t, err)
+
+	testCases := []struct {
+		params   []string
+		expected []byte
+		errMsg   string
+	}{
+		{params: []string{":child_storage_key", ""}, expected: nil},
+		{params: []string{":child_storage_key", ":child_first"}, expected: []byte(":child_first_value")},
+		// ??? trie with given root does not exist: 0xa5eb26ce5a6131f0737afbafa33ac3b9677a4fdb014d584862fe5da9d7d4539b
```

Comment:
This should be removed as it was only included for development purposes.

Reply:
removed!